### ISSUE TITLE
All Sims Games: Terrible FMV Intro Lag

### ISSUE THREAD

**Issue by ghyujkilop** [xemu-project/xemu#1744]
### Title

[The Urbz: Sims in the City](https://xemu.app/titles/45410060#The-Urbz-Sims-in-the-City)
[The Sims](https://xemu.app/titles/4541002f#The-Sims)
[The Sims 2](https://xemu.app/titles/4541038b#The-Sims-2)
[The Sims: Bustin' Out](https://xemu.app/titles/45410043#The-Sims-Bustin-Out)

### Bug Description

All FMV Intros on Xemu lag for all Sims games.

[Xemu - The Urbz: Sims in the City - FMV Intro Video Lag](https://youtu.be/Zk7IuywJ7so)
[Xemu - The Sims - FMV Intro Video Lag](https://youtu.be/a24krjt9Kjs)
[Xemu - The Sims 2 - FMV Intro Video Lag](https://youtu.be/Ktp6WR2LW9g)
[Xemu - The Sims: Bustin' Out - FMV Intro Video Lag](https://youtu.be/tnIpQCpEXBM)

### Expected Behavior

No lag on native.

### xemu Version

0.7.131

### System Information

CPU:          Intel(R) Core(TM) i7-9750H CPU @ 2.60GHz
OS Platform:  Windows
OS Version:    24H2
Manufacturer: NVIDIA Corporation
GPU Model:    NVIDIA GeForce RTX 2070/PCIe/SSE2
Driver:       4.0.0 NVIDIA 566.36
Shader:       4.00 NVIDIA via Cg compiler

### Additional Context

I have commented this on other threads, but figured it'd be easier to just have it all in one place.

**Comment by KreeseXD** [comment]
I don't get any intro lag for Sims 1 or 2, played fine without any audio issues; I idled on the main menu on Sims 1 and that idle FMV stuttered a bit, but not as bad as the youtube links you had

CPU:          13th Gen Intel(R) Core(TM) i9-13980HX
OS Platform:  Windows
OS Version:    24H2
Manufacturer: NVIDIA Corporation
GPU Model:    NVIDIA GeForce RTX 4080 Laptop GPU/PCIe/SSE2
Driver:       4.0.0 NVIDIA 566.14
Shader:       4.00 NVIDIA via Cg compiler

Version:      0.8.35
Branch:       master
Commit:       a00820746f7a6e3322a9ee2c471138d3791c2f32
Date:         Sat Mar 15 01:48:06 UTC 2025

Edit: added specs and version

**Comment by ghyujkilop** [comment]
> I don't get any intro lag for Sims 1 or 2, played fine without any audio issues; I idled on the main menu on Sims 1 and that idle FMV stuttered a bit, but not as bad as the youtube links you had
> 
> CPU: 13th Gen Intel(R) Core(TM) i9-13980HX OS Platform: Windows OS Version: 24H2 Manufacturer: NVIDIA Corporation GPU Model: NVIDIA GeForce RTX 4080 Laptop GPU/PCIe/SSE2 Driver: 4.0.0 NVIDIA 566.14 Shader: 4.00 NVIDIA via Cg compiler
> 
> Version: 0.8.35 Branch: master Commit: [a008207](https://github.com/xemu-project/xemu/commit/a00820746f7a6e3322a9ee2c471138d3791c2f32) Date: Sat Mar 15 01:48:06 UTC 2025
> 
> Edit: added specs and version

Yeah, unfortunately I tested both of those again, and still laggy intros. I hope my issue(s) won't get tied down to "bad hardware", because if I can run other emulators just fine, then how can it be my hardwares fault.

**Comment by ghyujkilop** [comment]
So, an update. I've now updated my hardware and software (switched to cachyos as well). And what's intetresting, is that all these intros are in opengl, but the backend that works is Vulkan for The Sims, and that's the only one that won't lag on the fmv. Anyway, closing this as it seems to mostly be resolved.
